src/server.js

```diff
@@ -92,9 +92,9 @@ module.exports = function (options = {}) {
 
 	let server = null;
 
-	if (Helper.config.public && (Helper.config.ldap || {}).enable) {
+	if (Helper.config.public && (Helper.config.ldap || Helper.config.headerAuth || {}).enable) {
```

Comment:
This isn't checking what we intend to be checking. As an example:

```js
const a = {enable: false};
const b = {enable: true};
console.log((a || b || {}).enable);
```

That will print false, even though `b` is enabled.